sys/vm/vm_mmap.c

```diff
@@ -679,16 +677,11 @@ kern_mmap_req(struct thread *td, const struct mmap_req *mrp)
 	 */
 	if (align == MAP_ALIGNED_CHERI) {
 		flags &= ~MAP_ALIGNMENT_MASK;
-		if (CHERI_REPRESENTABLE_ALIGNMENT(size) > (1UL << PAGE_SHIFT)) {
+		if (CHERI_REPRESENTABLE_ALIGNMENT(size) > PAGE_SIZE) {
 			flags |= MAP_ALIGNED(CHERI_ALIGN_SHIFT(size));
```

Comment:
It may be good to note in the commit message that we can get away with merely padding the length because the existing code already enforces sufficient alignment of the base.